packages/website/pages/docs/guides/run-a-node.mdx

```diff
@@ -40,14 +40,22 @@ Then, open the `.env` file in your preferred text editor:
   <Tab>```sh notepad .env ```</Tab>
 </Tabs>
 
-Finally, set the following environment variables to configure your node: 
+Finally, set the following environment variables:
```

Comment:
Also remind people that if they have run alpha-1 node before, they need `docker compose down -v`?

Reply:
addressed.